src/autolabel/models/openai.py

```diff
@@ -221,12 +226,27 @@ async def _alabel(self, prompts: List[str]) -> RefuelLLMResult:
             )
         except Exception as e:
             logger.exception(f"Unable to generate prediction: {e}")
+            error_message = str(e)
```

Comment:
Added new error parsing logic for OpenAI here. If we want to land these changes, can also add similar logic for other providers.